tests/utils_tests/test_dateformat.py

```diff
@@ -115,6 +115,10 @@ def test_dateformat(self):
     def test_day_of_year_leap(self):
         self.assertEqual(dateformat.format(datetime(2000, 12, 31), 'z'), '366')
 
+    def test_iso_year_edgecase(self):
+        d = datetime(2003, 12, 29)
+        self.assertEqual(dateformat.format(d, 'o'), '2004')
```

Comment:
This is already tested in `template_tests.filter_tests.test_date.DateTests.test_date04()/test_date05()`. I will remove this test.

Reply:
Oh, you're right! I didn't think to check for a test in a different folder.